.claude/skills/sync-shared-surfaces/SKILL.md

```
@@ -0,0 +1,217 @@
+git -C <editor> ls-tree origin/<branch> -- <path>
+git -C <web>    ls-tree origin/<branch> -- <path>
+```
+
+If those blob hashes match, the git index is already in sync and no action
+is needed — report it but don't copy.
+
+## Present the plan
+
+Before touching any file, show the user a summary:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)